src/components/Radio/index.js

```diff
@@ -87,9 +87,23 @@ const LabelText = glamorous.span({
   }
 });
 
-const Radio = ({ children, inline, input }) => (
+const Radio = ({
```

Comment:
I would categorise form components in to one of 2 types:

1. `Field` if it receives both input and meta data from final form, i.e. displays validation message
2. `Input` if it just receives input props

This seems to be an Input.

I would suggest all inputs should receive props directly rather than an `input` object prop.

i.e. parent field/group would pass `<Input {...input} />`